forge-gui/res/cardsfolder/upcoming/swine_rebellion.txt

```diff
@@ -0,0 +1,15 @@
+Name:Swine Rebellion
+ManaCost:2 G
+Types:Sorcery
+A:SP$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE3 | TrueSubAbility$ DBConjureAll | FalseSubAbility$ DBChoose | SpellDescription$ If you control three or more Boars with different names, conjure each card from the Three Pigs spellbook onto the battlefield.\nIf you control two or fewer Boars with different names, conjure two cards of your choice from the Three Pigs spellbook into your hand, then put one of them onto the battlefield.
+SVar:DBConjureAll:DB$ MakeCard | Conjure$ True | Names$ First Little Pig,Second Little Pig,Third Little Pig | Zone$ Battlefield 
```

Comment:
This part says "Conjure each"

Reply:
Yeah, `Names$` param is for all the cards and `Spellbook$` for one of the cards